wordpress-mobile/gutenberg-mobile
# Fix block insertion UI test

Gutenberg PR: https://github.com/WordPress/gutenberg/pull/31053

To test:

1. Execute the command `npm run test:e2e:ios:local` to run the UI tests
2. Observe that all of them pass (specifically `Gutenberg Editor tests for Block insertion 2`)

**This can also be verified by triggering the optional UI Tests in the PR checks.**

PR submission checklist:
